packages/fast-check/src/arbitrary/bigInt.ts

```diff
@@ -81,6 +81,15 @@ function bigInt(min: bigint, max: bigint): Arbitrary<bigint>;
  * @public
  */
 function bigInt(constraints: BigIntConstraints): Arbitrary<bigint>;
+/**
+ * For bigint between min (included) and max (included)
+ *
+ * @param args - Either min/max bounds as an object or constraints to apply when building instances
+ *
+ * @remarks Since 2.6.0
+ * @public
+ */
+function bigInt(...args: [] | [bigint, bigint] | [BigIntConstraints]): Arbitrary<bigint>;
```

Comment:
Should we replace the two overloads instead of being one extra one? I believe we could drop all overloads with this approach. That said not sure if we can have the same level of jsdoc with my suggestion.

Your suggestion has the benefit to be a first move. Let's do it that way for the first iteration.